CONTRIBUTING.md

```diff
@@ -163,7 +163,77 @@ in general try to follow them.
 - `replace` statements in `go.mod` files can be automatically inserted by running `make crosslink`. For more information
   on the `crosslink` tool see the README [here](https://github.com/open-telemetry/opentelemetry-go-build-tools/tree/main/crosslink).
 
-## Adding Labels via Comments
+## Issue Triaging
+
+To help provide a consistent process for seeing issues through to completion, this section details some guidelines and
+definitions to keep in mind when triaging issues.
+
+### Roles
+
+Determinining the root cause of issues is a shared responsibility between those with triager permissions, code owners,
+OpenTelemetry community members, and anyone else who would like to contribute.
+
+#### Triagers
+
+Contributors with [triager](https://github.com/open-telemetry/opentelemetry-collector-contrib/#contributing) permissions can help move issues along is through adding missing component labels,
```

Comment:
Wording is a little jumbled here

Reply:
Looks like I left a few things over from a previous re-wording, thanks.